addons/hr_homeworking/static/src/calendar/common/popover/calendar_common_popover.js

```diff
@@ -0,0 +1,32 @@
+/** @odoo-module **/
+
+import { patch } from "@web/core/utils/patch";
+import { AttendeeCalendarCommonPopover } from "@calendar/views/attendee_calendar/common/attendee_calendar_common_popover";
+import { Field } from "@web/views/fields/field"
+patch(AttendeeCalendarCommonPopover.prototype, "hr_homeworking_calendar_common_popover",{
+
+    setup() {
+        this.fieldNames = ["work_location_id", "work_location_name", "work_location_type", "employee_id", "weekday", "weekly", "start_date"];
+        this._super(...arguments)
+    },
+    isWorkLocationEvent(){
+        return this.props.record.rawRecord['resModel'] === 'hr.employee.location';
+    },
+    get isEventDeletable() {
+        return ('resModel' in this.props.record.rawRecord) || this._super(...arguments)
+    },
+    get displayAttendeeAnswerChoice() {
+        return (!'resModel' in this.props.record.rawRecord) && this._super(...arguments)
```

Comment:
/!\ `(!a in b)` != `!(a in b)` 

```suggestion
        return !('resModel' in this.props.record.rawRecord) && this._super(...arguments)
```